DistFiles/xMatter/bloom-xmatter-common.less

```diff
@@ -20,6 +20,9 @@
          //Just center the margin box, (for now, we're ignoring the binding) 
           left: @MarginOuter;
           width:  @PageWidth - (2 * @MarginOuter);
+          IMG { /* BL-2353 Keeps XMatter thumb images from going too wide (or too small) */
+              max-width: @PageWidth - (2 * @MarginOuter);
```

Comment:
I don't see why this one is 2*@MarginOuter rather than @MarginOuter + @MarginInner. I think a comment is indicated.
Is there another case for something like @MarginEven when they are balanced?